packages/cozy-procedures/src/redux/personalDataSlice.js

```diff
@@ -22,6 +52,37 @@ const selectors = {
 }
 
 const { actions, reducer } = personalDataSlice
-export const { init, update } = actions
+
+export const {
+  init,
+  update,
+  fetchMyselfLoading,
+  fetchMyselfSuccess,
+  fetchMyselfError
+} = actions
+
+export function fetchMyself(client) {
```

Comment:
I think we can add the client to the configuration. 

In drive with have something like :   const middlewares = [thunkMiddleware.withExtraArgument({ client, t })] to inject client in the thunk